### DISCUSSION TITLE
Massive list of new food ideas (apple trees?)

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#703] [Ideas]
The list of suggestions was getting a bit long so I decided to consolidate it into one post:

Egg Crates: Currently, eggs only stack to 16 and are extremely annoying to store in large amounts. Adding a crate would also mean this would work with autocrafting/compacting mods too.

Mushroom Crates: It would be cool if, since they can now be farmed just like the other crops, mushrooms could also be compacted into crates.

Apple Crate: for the same reason as above. I seem to have a lot and no way to compact them

Berry crates: sweet and maybe glow berry crates too? This one is more gimmicky and less of a necessity compared to the others.

Prime Rib: Unlike the pig, the cow is noticeably lacking a special drop, and as such, there appears to be no reason to kill it with a knife. I understand your intent was to make the pig worth farming, so just make these ribs very rare (10%)? They would not be made into a multiuse platter like the glazed ham, but could instead be made into a "steak dinner" plate, the best food item for raw hunger, but of course, only single use unlike the ham.

Drumstick: While you're at it, a "drumstick" drop from chickens too (max of two at a time)? Can be cooked into "fried chicken" then 3 crafted into a "bucket of chicken" (multiuse? but again, not as good as the ham). I guess what im saying is all animals should have a reason to kill it with the knife, even if you still want the pig to be the best.

Ice Cream: Milk, Ice, sugar in a bowl should make ice cream. Perhaps you could optionally add a cocoa bean to make it chocolate ice cream instead (they would be functionally identical and this would solely be a cosmetic change).

Grapes: would be the first crop in your mod to hang from blocks and would periodically grow grapes similar to glow berries. They could be used to make grape juice, a grape popsicle, grape jelly (which could be put on toast, see below), or perhaps wine (that gives you exponentially worsening nausea if you drink more than one in a short period of time)? Also, I dont think a grape pie would make sense, but according to google its a thing so up to you. Would also be included/replacement item for the existing fruit salad recipe? They could be cooked to make raisins, which could then be used for MORE recipes like raisin bread (made from raisin dough), raisin cookies, oatmeal? etc

Toast: Please have bread be able to be cut into 4 toast (they would be used in sandwich recipes instead of bread loaves, and 4 becuase you would need 2 toast per sandwich instead of 1). It could also be used in new recipes like toast and jelly (with grape jelly from above) and raisin bread would also be able to be cut into raisin toast.

Burnt Bread: Cooking bread/toast again would result in burnt bread which can no longer be used in sandwich recipes. Why would you want to do this? idk but it would make sense

Golden Delight: There are currently no recipes that use the golden apple, and the only one that uses golden carrots is for horses. This would also be nod to real life gold plated foods, and golden apples could perhaps be included in a "golden fruit salad" that gives regen for longer?

Apples: Applesauce bowl and maybe Honey'd apples? In my experience it is difficult to use apples in bulk, so I was looking for a simpler recipe.

Apple slices: 4 per apple (1 hunger/1 saturation each), eaten fast like kelp? I suppose none of the existing recipes would make sense to use slices instead of full apples, but it could be a fun gimmick addition.

Apple Trees: Cutting an apple also yields 1-2 apple seeds which could then be planted as a crop. After a while it grows into an apple sapling, and then finally into an apple tree (similar to avocado from cultural delight). Apple trees would be a much easier and reliable way of getting large amounts of apples for pies/cider (as of now, they are the only food ingredient that cannot be easily farmed. Even mushrooms now can on rich soil). These trees however would only be grown from seeds and would not drop the saplings directly like other trees. Perhaps eating any apple yields a core that can be crafted into 1 apple seeds (so its better to cut if you need seeds)

Omelet: A cooking recipe that uses eggs, perhaps requires milk bottle, tomato, and maybe bacon? Or that could be a separate "bacon omelet" recipe?

French fries: 2 yielded from cutting potatoes, which can then be cooked. Eaten fast like kelp?

**Comment by thelegitdolt** [comment]
nice

**Reply by Aceplante** [reply]
Thanks! what parts did you like the most?

**Reply by thelegitdolt** [reply]
i would love me some golden fruit salads

**Comment by BarchamMal** [comment]
> Egg Crates: Currently, eggs only stack to 16 and are extremely annoying to store in large amounts. Adding a crate would also mean this would work with autocrafting/compacting mods too.
> ...

~~As you suggest 4 different crates, you may want to split this discussion.~~
Also, I totally think about everything here should be added, except for burnt bread (then what about burnt apples, or steak, or potatoes)

**Comment by MythrilBagels** [comment]
I can certainly understand the crates suggestions, but I have a few nitpicks with some of the other ideas.

For instance, your idea for Prime Ribs is flawed in a few areas. Your argument for why is that "The cow is noticeably lacking a special drop", yet you completely ignore the existence of leather, which cows will drop more of if killed with a knife. Also, having a single Cooked Rib give the Nourishment effect would likely be unbalanced, and it would go against the precedent of plated meals giving the effect. There is a pattern to Vector's madness lol
Your idea for "Chicken Drumsticks" is also unnecessary, as that is essentially what Chicken Cuts are. If there was Fried Chicken (something I'm doing in one of my own addons) it would likely use those instead.
Vector has previously stated that there will not be toast, as there is already a method of obtaining more bread per wheat, through the use of dough.
As for burnt bread... no, just no

A lot of your other suggestions are coincidentally features I plan on adding to some of my own concepts (such as Fried Chicken, Fries, Apple Trees, and potentially Grapes but I'm not certain of that one yet.

**Reply by BarchamMal** [reply]
Wow, didn't think about leather, and forgot that @Aceplante2 asked for chicken drumsticks. I agree with you there.
As for Cooked Rib, it could be a plated food that you can only get like two servings from, thus keeping pigs more worth farming than cows.

**Reply by Aceplante** [reply]
> Wow, didn't think about leather, and forgot that @Aceplante2 asked for chicken drumsticks. I agree with you there. As for Cooked Rib, it could be a plated food that you can only get like two servings from, thus keeping pigs more worth farming than cows.

yea that's basically what I was thinking. When I initially proposed the prime rib, I was told he did not plan to add a special drop like that becuase he wanted to make the pig the best. And I sort of agree; in vanilla, the cow is literally a straight upgrade to the pig, the drops giving the same hunger but the cow also dropping leather. So having only the pig have a special drop sort of solves this. But, I still think a special "prime rib" drop would be cool. So, instead of being made into a multiuse platter, it would simply be a really good food source on its own? Perhaps by only being made into a single use "steak dinner" plate, unlike the ham which of course can be made into a multi use platter, but would be the best raw hunger food item?
